src/angular/6-restaurant-service/restaurant.service-1.ts

```diff
@@ -1,15 +1,15 @@
-import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
 import { environment } from '../../environments/environment';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class RestaurantService {
+  constructor(private httpClient: HttpClient) {}
 
-  constructor(private httpClient: HttpClient) { }
-
-  getRestaurants() {
+  getRestaurants(): Observable<any> {
```

Comment:
can we type the any as Restaurant[] right?

Reply:
whoops - is this before we add Types?

Reply:
yes, before adding Restaurant type :)